catalog-info.yaml

```diff
@@ -18,19 +22,97 @@ metadata:
     - title: Logstash filter for Elastic integration
       url: https://github.com/elastic/logstash-filter-elastic_integration
 spec:
+  type: tool
+  owner: group:ingest-fp
+  system: platform-ingest
+  lifecycle: production
+
+# ***********************************
+# SECTION START: Pull Request pipeline
+# ***********************************
+
+---
+# yaml-language-server: $schema=https://gist.githubusercontent.com/elasticmachine/988b80dae436cafea07d9a4a460a011d/raw/rre.schema.json
+apiVersion: backstage.io/v1alpha1
+kind: Resource
+metadata:
+  name: logstash-filter-elastic-integration-pull-request-pipeline
+  description: 'A pipeline for Logstash filter for Elastic integration Pull Requests'
+  links:
+    - title: 'Logstash filter for Elastic integration Pull Request pipeline'
+      url: https://buildkite.com/elastic/logstash-filter-elastic-integration-pull-request-pipeline
+spec:
+  owner: group:ingest-fp
+  type: buildkite-pipeline
+  system: buildkite
   implementation:
     apiVersion: buildkite.elastic.dev/v1
     kind: Pipeline
     metadata:
-      description: ':logstash: The logstash-filter-elastic_integration :pipeline:'
-      name: logstash-filter-elastic_integration-ci
+      description: ':logstash: The logstash-filter-elastic_integration-pull-request :pipeline:'
+      name: logstash-filter-elastic_integration-pull-request
     spec:
-      pipeline_file: .buildkite/pipeline.yml
-      provider_settings:
-        build_pull_request_forks: true
+      pipeline_file: .buildkite/pull-request-pipeline.yml
```

Comment:
let's add a maximum timeout time, otherwise if it hangs, it can stay stuck up to 24hrs which is the default. Timeout can be defined at the `implementation.spec:` level, with:

```yaml
spec:
  implementation:
    spec:
      pipeline_file: ...
      maximum_timeout_in_minutes: 120
```